packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -211,6 +212,13 @@ export class WorkflowWrapper {
 	}
 
 	addNode(op: Operation, pos: Position, options: { size?: OperatorNodeSize; state?: any }) {
+		let currentUserName: string | undefined = '';
+		try {
+			currentUserName = useAuthStore().user?.username;
```

Comment:
Using the initials like we do for the user menu would make a lot more sense.
```suggestion
			currentUserName = useAuthStore().userInitials;
```

Reply:
If the goal is to keep track of which user and not for displaying, you should use the id.

Reply:
You are not wrong, but between workflowId, nodeId, source, target, active-output, source-port, target-port and a few others, that is a whole lot of uuids to track and having one less helps.